.husky/pre-push

```diff
@@ -1,6 +1,4 @@
-#!/usr/bin/env sh
-
 pnpm check-lock-files &&
-pnpm check-commit &&
-pnpm documentation &&
-pnpm pretty-quick --check
+    pnpm check-commit &&
```

Comment:
Why this indentation?

Reply:
Why do we even have ampersands here? 🤔 

Let's remove them and just have individual lines I think

Reply:
Done